### DISCUSSION TITLE
Unable to import module 'serverless_aws_lambda_sdk.internal_extension.wrapper'

### DISCUSSION THREAD

**Discussion by gCampiani** [serverless/serverless#12301] [General]
After I enabled "Instrument" in the serverless console it added a lambda layer to my function. It seems fine until I tried to run my API (using FastAPI) and it fails with the following message:

`[ERROR] Runtime.ImportModuleError: Unable to import module 'serverless_aws_lambda_sdk.internal_extension.wrapper': cannot import name 'Doc' from 'typing_extensions' (/opt/python/typing_extensions.py)`

Everything is working fine until I try to instrument my function, here's what I tried before comming here:

- Tried to manually install `serverless-sdk = "^0.6.0"`
- Tried to manually install `typing-extensions = "^4.9.0"`
- Tried to remove the layer with the instrument on `sls-sdk-python-v0-3-2` (Obviusly didnt work and got the error `/opt/sls-sdk-python/exec_wrapper.py: does not exist`)

It's good to point that all my functions are working correctly, except for this one with FastAPI framework and I haven't found anything about it at Google or GPT4

**Comment by Verde1705** [comment]
I don't know if it's too late, my solution so far for this exact error is to change the loading order of the layers, since the sdk seems to have a lower version of the one used by fastapi, anyway, if you use fastapi 0.103 or lower, you should not have this error
